tests/kafkatest/services/kafka/kafka.py

```diff
@@ -165,6 +165,19 @@ def start(self, add_principals=""):
         self.start_minikdc(add_principals)
         Service.start(self)
 
+        self.logger.info("Waiting for brokers to register at ZK")
+
+        cmd = self.path.script("zookeeper-shell.sh", self.zk) + " " + self.zk_connect_setting() + " ls /brokers/ids"
+
+        broker_ids = ""
```

Comment:
this looks cleaner as `"[" + ", ".join(self.idx(node) for node in self.nodes)` + "]"